velox/vector/DecodedVector.cpp

```diff
@@ -514,4 +516,35 @@ VectorPtr DecodedVector::wrap(
       rows.end(),
       std::move(data));
 }
+
+const uint64_t* DecodedVector::nulls() {
```

Comment:
These changes probably belong in a different PR ?

Reply:
Indeed. See my reply above.